src/lib/project/twitter.ex

```diff
@@ -0,0 +1,253 @@
+defmodule Project.Twitter do
+  @moduledoc """
+  The Twitter context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Project.Repo
+
+  alias Project.Twitter.Post
+  alias Project.Twitter.User
+
+  @doc """
+  Returns the list of posts.
+
+  ## Examples
+
+      iex> list_posts()
+      [%Post{}, ...]
+
+  """
+  @spec list_posts :: [%Post{}]
```

Comment:
せっかく`@type`で作ったタイプを使ったください。Userも同様です。
```suggestion
  @spec list_posts :: list(Post.t())
```